src/components/home.vue

```diff
@@ -1,21 +1,362 @@
+            this.mostPopular = data.results;
+          })
+      )
+      .then(
+        fetch(
+          "https://api.themoviedb.org/3/tv/airing_today?api_key=ffebf14b46dcd2b2bb0af17fdfffaa0c"
+        )
+          .then((data) => {
+            return data.json();
+          })
+          .then((data) => {
+            this.nowPlaying = data.results;
+          })
+      )
+      .then(
+        fetch(
+          "https://api.themoviedb.org/3/tv/on_the_air?api_key=ffebf14b46dcd2b2bb0af17fdfffaa0c"
+        )
+          .then((data) => {
+            return data.json();
+          })
+          .then((data) => {
+            this.upcoming = data.results;
+          })
+      )
+      .then(
+        fetch(
+          "https://api.themoviedb.org/3/tv/top_rated?api_key=ffebf14b46dcd2b2bb0af17fdfffaa0c"
+        )
+          .then((data) => {
+            return data.json();
+          })
+          .then((data) => {
+            this.topRated = data.results;
+          })
+      );
+  },
+};
 </script>
 <style scoped>
+.wrapper {
+  height: 480px;
+  margin: 17px 0 -17px 0;
+  display: flex;
+  overflow-x: auto;
+  padding: 41px;
+  background-color: #f8f0fb;
+}
+
+.wrapper img {
+  width: 150px;
+  height: 225px;
+  display: inline-block;
+  padding: 10px;
+  border-radius: 20px;
+}
+
+.wrapper p {
+  text-align: center;
+  font-family: "Franklin Gothic Medium", "Arial Narrow", Arial, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 20px;
+}
+
+h3 {
+  display: inline-block;
+  margin: 70px 0 0 0;
+  padding: 0 40px 0 0;
+}
+.old {
+  background-color: white;
+  border-radius: 10px;
+  height: 330px;
+  width: 152px;
+  margin: 25px 5px;
+  border: 1px solid #ccc;
+}
+
+.old:hover {
+  margin-top: -30px;
+  margin-left: 30px;
+  margin-right: 30px;
+  max-height: 400px;
+  width: 170px;
+}
+.old:hover img {
+  width: 170px;
+  height: 255px;
+}
+
+hr {
+  padding: 2px;
+}
+.button {
+  position: absolute;
+  height: 30px;
+  left: 290px;
+  top: 133px;
+}
+button {
+  border: 1px solid #032541;
+  border-radius: 30px;
+  width: 150px;
+  background-color: white;
+  margin-left: 7px;
+}
+.available #movies {
+  background-color: #032541;
+  color: white;
+}
+.notAvailable #tvShows {
+  background-color: #032541;
+  color: white;
+}
+.button1 {
+  position: absolute;
+  height: 30px;
+  left: 290px;
+  top: 750px;
+}
+.available1 #movies1 {
+  background-color: #032541;
+  color: white;
+}
+.notAvailable1 #tvShows1 {
+  background-color: #032541;
+  color: white;
+}
+.button2 {
+  position: absolute;
+  height: 30px;
+  left: 290px;
+  top: 1375px;
+}
+.available2 #movies2 {
+  background-color: #032541;
+  color: white;
+}
+.notAvailable2 #tvShows2 {
+  background-color: #032541;
+  color: white;
+}
+.button3 {
+  position: absolute;
+  height: 30px;
+  left: 290px;
+  top: 1992px;
+}
+.available3 #movies3 {
+  background-color: #032541;
+  color: white;
+}
+.notAvailable3 #tvShows3 {
```

Comment:
You can see that all classes like notAvailable2, available2 are same. Please remove them and use a single class ;)